models/articles.models.js

```diff
@@ -35,3 +35,24 @@ exports.selectArticles = () => {
       }
     });
   };
+exports.patchArticle = (article_id, patchedArticle) => {
+    const { inc_votes } = patchedArticle;
+    const queryStr = format(`UPDATE articles SET votes = votes + %L WHERE article_id = %L RETURNING *;`, inc_votes, article_id);
```

Comment:
I think with this you could use the default `$1` and `$2` binders, meaning you could skip having to use `format` - but this also works fine